sycl/test-e2e/ESIMD/regression/atomic_update_test.cpp

```diff
@@ -35,26 +37,28 @@ int main() {
                                                                      0, 1)[0];
       sycl::ext::intel::experimental::esimd::lsc_atomic_update<
           sycl::ext::intel::esimd::atomic_op::store, uint32_t, 1>(
-          int_sync, 1, int_result, 1);

```

Comment:
I just realized something. The operand '1' that goes right after 'int_sync' is the vector offsets, right?
It may be wrong because 1 means 1-byte offset, while 'int_sync' is 4-byte element and the offset parameter must be 4, not 1.

If replacing '1' with '4' here will help to stabilize the test, then it would be better not to split input and output. Because the test "before" tests the offset parameter better than just passing 0 as in the proposed variant now.
Instead of separating input and output the test